tinygrad/ops_gpu.py

```diff
@@ -121,7 +122,8 @@ def backward(ctx, grad_output):
 
 class Max(Function):
   def forward(ctx, input, axis=None):
-    axis = [axis] if type(axis) == int else axis
```

Comment:
I prefer inline here

Reply:
do u prefer `if isinstance(axis, int): axis = [axis]` or `axis = [axis] if isinstance(axis, int) else axis`?
i choose first cuz `axis = axis` is nasty but will comply to your preferences for future commits